### ISSUE TITLE
Naming consistency issues

### ISSUE THREAD

**Issue by soniakeys** [soniakeys/graph#28]
Types:

    LabeledAdjacencyList  DirectedLabeled  UndirectedLabeled
           AdjacencyList  Directed         Undirected

Maybe AdjacencyListLabeled?

Identifiers:  Use `ma` for arc size; m is misleading. -- fixed

Use consistent name for FromList result in various search structs  (merged from #20)
Currently:
AStar.Tree
BellmanFord.Tree
BreadthFirst.Result
BreadthFirst2.Result
DAGPath.Tree
Dijkstra.Tree
Prim.Forest

**Comment by soniakeys** [comment]
Nn, didn't mean to close it quite yet.

So, it's complicated.  Of course a FromList encodes a forest, so "Result" is a descriptive default.  Unless it's in a struct with other result members.  Then "Forest" is next best.  Unless there is a guarantee that there is a single root, then "Tree" is more accurate and less confusing.  Unless there is some other forest or tree in the struct, then ResultForest...

action: changed existing Trees to Forests, left BF searches with Results.

**Comment by soniakeys** [comment]
Still don't like the inconsistent type names.  Leaning toward AdjacencyListLabeled at the moment.

**Comment by soniakeys** [comment]
So, I actually tried AdjacencyListLabeled, and turned around some other things as well to the military-like <noun> <adjective> style but didn't like it.  I slightly more liked the way it ordered things, but much disliked the awkward readability.  So instead I changed DirectedLabeled to LabeledDirected and committed.

I combined this change with a big file contents cleanup since both were going to touch lots of lines.
